antarest/launcher/model.py

```diff
@@ -324,3 +344,256 @@ class LauncherLoadDTO(AntaresBaseModel, extra="forbid", alias_generator=to_camel
+                raise BadLauncherConfigInput("min_antares_version cannot be greater than max_antares_version")
+
+        requires_version_9_2 = (
+            self.min_antares_version is not None and self.min_antares_version >= MIN_SUPPORTED_VERSION_FOR_OPTIM_PARAMS
+        )
+
+        # linear_solver_param_optim_* only valid for >= 9.2
+        if self.linear_solver_param_optim_1 and not requires_version_9_2:
+            raise BadLauncherConfigInput(
+                "linear_solver_param_optim_1 is not supported before Antares version 9.2 "
+                f"(got {self.min_antares_version})"
+            )
+
+        if self.linear_solver_param_optim_2 and not requires_version_9_2:
+            raise BadLauncherConfigInput(
+                "linear_solver_param_optim_2 is not supported before Antares version 9.2 "
+                f"(got {self.min_antares_version})"
+            )
+
+        return self
+
+
+def make_other_options_from_launcher_config(config: LauncherConfigDTO) -> str:
+    """
+    Generate an 'other_options' string from a LauncherConfigDTO. This will be passed to the antares launcher script
+
+    Example output:
+        --solver=xpress --nobasis1 --param-optim1="THREADS 4 PRESOLVE 1" --param-optim2="MIPRELSTOP 0.01"
+    """
+    options: List[str] = []
+
+    solver = config.linear_solver.lower()
+    options.append(f"--solver={solver}")
+
+    if not config.use_optim_1_basis_next_week:
+        options.append("--nobasis1")
+    if not config.use_optim_1_basis_optim_2:
+        options.append("--nobasis2")
+
+    # Build per-optim strings
+    def build_param_str(param_list: List[Tuple[str, str]]) -> str:
+        return " ".join(f"{k} {v}" for k, v in param_list)
+
+    # param-optim1
+    param1 = build_param_str(config.linear_solver_param_optim_1 or [])
+    param_common = build_param_str(config.linear_solver_param or [])
+    combined1 = f"{param_common} {param1}".strip()
+    if combined1:
+        options.append(f'--param-optim1="{combined1}"')
+
+    # param-optim2
+    param2 = build_param_str(config.linear_solver_param_optim_2 or [])
+    combined2 = f"{param_common} {param2}".strip()
+    if combined2:
+        options.append(f'--param-optim2="{combined2}"')
+
+    return " ".join(options)
+
+
+def overwrite_params_other_options_with_config(
+    launcher_params: LauncherParametersDTO,
+    launcher_config: LauncherConfigDTO,
+) -> None:
+    new_other_options = make_other_options_from_launcher_config(launcher_config)
+    launcher_params.other_options = new_other_options
+
+
+class LauncherConfigModel(Base):
+    __tablename__ = "launcher_configuration"
+
+    id = mapped_column(String(36), primary_key=True)
+    name = mapped_column(String)
+    linear_solver = mapped_column(String)
+    min_antares_version = mapped_column(JSON, nullable=True)
+    max_antares_version = mapped_column(JSON, nullable=True)
+    linear_solver_param_optim_1 = mapped_column(JSON, nullable=True)
```

Comment:
As commented in alembic, it's probably simpler to use a string here that we will parse.